PHPUnit Tests

dpDesignz opened the pull request:
Resolves #200 and also fixed phpunit error `expectExceptionMessageRegExp() is deprecated in PHPUnit 8 and will be removed in PHPUnit 9. Use expectExceptionMessageMatches() instead.`

techno-express commented:
Refresh/sync the changes in .travis.yml and composer.json, i fixed the configuration issues causing build stop errors. 

I see your changes causing some fail tests.

Also the phpunit warning messages is not a big issue, fixing them might break other version tests, cause they might not be available in lower **phpunit** or **php versions**, and only worth it if **phpunit 9** is force by some future version of PHP. The changes can stay as you have if not breaking other versions.

dpDesignz commented:
> Refresh/sync the changes in .travis.yml and composer.json, i fixed the configuration issues causing build stop errors. 
> 
> I see your changes causing some fail tests.
> 
> Also the phpunit warning messages is not a big issue, fixing them might break other version tests, cause they might not be available in lower **phpunit** or **php versions**, and only worth it if **phpunit 9** is force by some future version of PHP. The changes can stay as you have if not breaking other versions.

How do you mean refresh/sync the changes? I'm not sure how to do that? My repo was up to date with the master according to github?

The failed tests seemed to be triggered from old files, because the lines that failed don't exist anymore. Thought it was a bit weird.

The functions that I changed them to have been available since 2013 from what I could see in the docs, so they shouldn't break anything :smile:

techno-express commented:
I meant pull sync with master branch again, i just found out that my push/update actually cause your PR to run again. 

Just check the tests results on travis ci you will see what i mean for failing tests. 
Try adding drop tablets before each of those tests, or use totally different table/names instead, they could be from other tests causing issue.
